templates/src/graphql/default.graphql

```diff
@@ -0,0 +1,44 @@
+"""
+This schema is the default graphql schema, please don't remove it.
+"""
+
+type User {
+  "The user id from the authentication middleware"
+  id: ID!
+  email: String!
+}
+"Presigned url object"
+type SignedUrl {
+  """
+  A presigned url generated by a cloud storage points to a specific file 
+  that has been stored or will be stored on a cloud storage
+  """
+  url: String
+  "The http request method such as GET, PUT, POST"
+  method: String 
+}
+
+"""
+A presigned urls collection that contains all presigned urls 
+for a specific file on a cloud storage 
+"""
+type SignedUrls {
+  "The presigned url object for file uploading"
+  upload: SignedUrl
+  "The presigned url object for file downloading"
+  download: SignedUrl
+}
+
+"""
+Type Query defines all query / read methods that clients can call. 
+These methods must be implemented in resolver as well.
+"""
+type Query {
+  """
+  Get all presigned urls for a specific file 
+  that has been stored or will be stored on a cloud storage
+  """
+  signedUrls(key: String!): SignedUrls

```

Comment:
I don't think there is going to be a common use case to get both of these at the same time. Mind splitting it into two queries?

Reply:
split query signedUrls to downloadSignedUrl and uploadSignedUrl. Thanks.